common/shared.c

```diff
@@ -893,14 +915,16 @@ static int ft_get_cq_comp(struct fid_cq *cq, uint64_t *cur,
 
 int ft_get_rx_comp(uint64_t total)
 {
-	int ret;
+	int ret = FI_SUCCESS;
 
 	if (rxcq) {
 		ret = ft_get_cq_comp(rxcq, &rx_cq_cntr, total, timeout);
 	} else {
-		ret = fi_cntr_wait(rxcntr, total, timeout);
-		if (ret)
-			FT_PRINTERR("fi_cntr_wait", ret);
+		while(fi_cntr_read(rxcntr) < total && ret == FI_SUCCESS) {
```

Comment:
Please fix spacing:  while (fi_cntr_read(rxcntr) < total) {
We can drop the ret == FI_SUCCESS check and break out of the loop inside the if statement to avoid checking the ret value twice.